### ISSUE TITLE
Settings are hard to find

### ISSUE THREAD

**Issue by seekr** [8bitPit/Niagara-Issues#342]
Hi, id like to start by saying I find Niagara great!

I found it easy to onboard and easy to do the main customisation (adding favourites). But... After almost a week using it only today I discovered how to get to settings (by search telegram group).

I think there should be a easier way to summon settings. Possibly long tapping area to the right of clock?

**Comment by Rod911** [comment]
Well the settings will always be at the bottom of the app list, once you get used to it it's only a swipe away.

**Comment by 8bitPit** [comment]
Thanks @seekr, you're right at least I should add a tutorial text.

**Comment by seekr** [comment]
@Rod911 I don't disagree that once I know it, it becomes easy to get to, but it's not very discoverable as it is. 

Maybe a "cog" icon after letter Z? (A, B, C, ..., X, Z, Cog) I think this would be good. A tutorial text may be a good complement.

Great, great launcher, though. Honestly.

**Comment by Rod911** [comment]
Agreed

**Comment by cauerego** [comment]
perhaps a cool way to make the "tutorial" would be when people would tap hold on the background the screen would smoothly move to the bottom and blink the settings button there.

then there would be an option to reactivate this behaviour under settings, which would be disabled once we enter settings for the first time. mostly just for fun.

**Comment by stalloone** [comment]
Also a quick tip: you don't even have to swipe for the settings. A quick tap just in the low right corner of the screen (below google button) will take you instantly to the very end of the app list. At least that's how it works for me!

+1 for the tutorial or some info after you install the launcher

**Comment by McKean** [comment]
Is there a possibility to define apps as "setting applications" showing them in the bottom area? Similar as defining a favorite. Not sure if this is already possible but IMHO should be.

**Comment by 8bitPit** [comment]
@McKean I would recommend opening system settings by swiping down to open the notification tray.

**Comment by McKean** [comment]
@8bitPit yeah that definitely works. But then again wouldn't it be cool to be able to define eg. Play store a password manager and settings as "settings apps" so they could enjoy the bottom part together with the Niagara settings?

**Comment by 8bitPit** [comment]
> to define eg. Play store a password manager

You mean as a default store application? Having to icons in the app list which do the same could feel redundant. But you're right, showing system settings at the bottom could work well nevertheless, I will test that.
Btw, #145 would be an alternate approach for opening the Store.

**Comment by Rod911** [comment]
I think you misunderstood what his requst was, I believe he wanted an option like 'add to settings' or something which would add it to the bottom of the app list along with Niagara settings and wallpapers. Did I get it right  @McKean ?

**Comment by McKean** [comment]
@Rod911 yeah, that's what I mean, but I think @8bitPit partially caught what I was trying to explain...  
In short it's just an inverted favorites for settings apps. Meaning the user can define as many settings apps that he wants and they'll be at the bottom of the list.

**Comment by 8bitPit** [comment]
@seekr & @cauerego A few versions ago, I made it easier to find settings by:
1. Showing the dot/ring (the settings section icon) always (it was only visible while scrolling before) 
2. Replacing the icon with a cog wheel if the user didn't open settings at all.
What do you think?

**Comment by 8bitPit** [comment]
@McKean Okay, then I misunderstood you. In 2019, I'll add folders to Niagara (#13). I'd suggest to continue the discussion once basic categorization features got implemented.

**Comment by cauerego** [comment]
much better than my idea! why not just keeping the cog instead of a ring there?

**Comment by 8bitPit** [comment]
@seekr & @cauerego Thanks for your replies. 
@cauerego in my opionion, the cog wheel doesn't fit very well, the alphabet looks unbalanced with it. The dot/ring is less distracting and adds a nice touch.
Here's a side by side comparison:

Cog |  Dot
:-------------------------:|:-------------------------:
![screenshot_20181221-232909](https://user-images.githubusercontent.com/9284845/50366151-66c65880-0578-11e9-8c01-9d23490c52f5.png) | ![screenshot_20181221-232930](https://user-images.githubusercontent.com/9284845/50366152-66c65880-0578-11e9-9bb1-3dde9d5082d8.png)
